README.md

```diff
@@ -109,3 +71,41 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a file called `<filename.html>` by default.
+This can be overwritten with the `-o` or `--output` option:
+
+```shell
+$ pyscript create --wrap <filename.py> -o <another_filename.html>
+```
+
+- ##### Open the script inside the browser using the `--show` option
+
+```shell
+$ pyscript create --wrap <filename.py> --show
+```
```

Comment:
-1 on this. This should be an option under the `run` cmd and if it means we are just showing the file, without running a static server it'll be prone to a bunch of possible errors that would probably frustrate users

Reply:
This is also borrowed from the previous functionality of `pyscript wrap`. It only shows the file on the browser yes, without the server.

Now that you mention this, I think we can remove this `--show` option altogether and let `pyscript run` handle this, as it does already.

Reply:
Removed `show` altogether as discussed.